src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -109,7 +109,11 @@ def _update_request_data_increased_size_limit(http_info: dict, max_size: int) ->
         return
     http_info["request"].update(
         {
-            "body": lumigo_dumps(HttpState.previous_request.body, max_size)
+            "body": lumigo_dumps(
+                HttpState.previous_request.body,
+                max_size,
+                omit_skip_path=HttpState.previous_request.omit_skip_path,
```

Comment:
if you'll put the `omit_skip_path` on the state, then this line will make much more sense! :)